src/components/Header/MobileHeader.js

```diff
@@ -192,19 +192,16 @@ class MobileHeader extends React.Component {
   */
   renderMyNyplButton() {
     let myNyplClass = '';
-    const gaAction = (this.props.patronName) ? 'MyAccount' : 'LogIn';
-    let icon = <LoginIcon className="loginIcon" ariaHidden focusable={false} />;
-    if (this.props.patronName) {
-      icon = (
-        <LoginIconSolid
-          className="loginIcon-loggedIn animated fadeIn"
-          ariaHidden
-          focusable={false}
-        />
-      );
-    }
+    const gaAction = 'MyAccount';
+    let icon = (
+      <LoginIconSolid
+        className="loginIcon-loggedIn"
```

Comment:
This was the one slight functional difference I see with the 16 update - the removal of the "animated" and "fadeIn" classes. I assume that was intentional

Reply:
Good catch, thanks. Yes, the removal of those classes are intentional and I'll update the React 16 branch.

Reply:
Updated.